lib/views/admin_request/hidden_user_explanation/_personal_information.text.erb

```diff
@@ -1,3 +1,20 @@
 <%= _('We consider it is not a valid FOI request as it was correspondence ' \
       'about your personal circumstances, and we have therefore hidden it ' \
       'from other users.') %>
+
+<%= _('* If you would like to know what information a public authority holds ' \
+      'about you, you should make a “Subject Access Request” in private ' \
+      'Data Protection law. The Information Commissioner’s website provides '\
```

Comment:
This should have been "_using_ Data Protection law", but I don't think this is relevant now that GDPR is in force, as noted in https://github.com/mysociety/alaveteli/issues/881#issuecomment-364785964.

I'd probably reduce this to:

```
…you should make a “Subject Access Request” in private. The Information Commissioner's website…
```